### PR TITLE
MGMT-10597: Open firewall ports after DNS setup.

### PR DESCRIPTION
**Pull request by paul-maidment**
We need to make sure that the appropriate ports have been exposed for
the services that have been exposed via libvirt DNS.

This is to ensure the assisted-service is able to connect to the API
server.

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [x] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @omertuc 
/cc @osherdp 

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: deploy/operator/ztp/deploy_spoke_cluster.sh [hunk 1]
```diff
@@ -136,3 +136,4 @@ if [ ${SPOKE_CONTROLPLANE_AGENTS} -eq 1 ] ; then
 fi
 
 setup_libvirt_dns
+open_firewall_ports
```
Comment (osherdp) [inline deploy/operator/ztp/deploy_spoke_cluster.sh:139]: @paul-maidment out of curiosity, what job / flow did it fix?

Comment (omertuc) [reply deploy/operator/ztp/deploy_spoke_cluster.sh:139]: It doesn't exist yet https://github.com/openshift/release/pull/28958

**Pull request merged**